### PR TITLE
LPS-150420 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-150420

@Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ozysouza** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 1]
```diff
@@ -670,85 +670,85 @@ definition {
...
 	@priority = "3"
 	test ViewLocalizedFriendlyURLInWCD {
-		task ("Add a display page template with Web Content Article and Basic Web Content") {
-			JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
-				contentType = "Web Content Article",
-				displayPageTemplateEntryName = "Display Page Name",
-				groupName = "Test Site Name",
-				subType = "Basic Web Content");
-		}
+		task ("Given a site designer has a web content with selected custom template, display page template and translated friendly URL") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:673]: Given a site designer has a web content with an associated display page template and a template with Friendly URL field

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase [hunk 2]
```diff
@@ -670,85 +670,85 @@ definition {
...
+					value1 = "webcontentspanish");
 
-		task ("Publish the web content article") {
-			PortletEntry.publish();
+				PortletEntry.publish();
+			}
 		}
 
-		task ("Add a Web Content Display and a Language Selector to widget page") {
+		task ("When the site designer displays the web content in Web Content Display widget") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase:751]: When the site designer displays the web content in Web Content Display with custom template

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, there are small changes

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/pull/8273 @ozysouza

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**